vendor/github.com/openshift/api/operator/v1/types_network.go

```diff
@@ -111,6 +111,10 @@ type AdditionalNetworkDefinition struct {
 	// rawCNIConfig is the raw CNI configuration json to create in the
 	// NetworkAttachmentDefinition CRD
 	RawCNIConfig string `json:"rawCNIConfig"`
+
+	// SRIOVCNIConfig is the OpenShiftSRIOV CNI configuration json to create in the
+	// NetworkAttachmentDefinition CRD
+	SRIOVCNIConfig string `json:"SRIOVCNIConfig"`
```

Comment:
You should create a `*struct SRIOVConfig` with a single member ` CNIConfig`

Reply:
Done